src/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/DeleteResourcesValidation.feature

```diff
@@ -42,21 +42,21 @@ Feature: Resources "Delete" Operation validations
 
 
         Scenario: 05 Verify response when deleting a referenced school
-             Given the system has these "schools" references
+            Given the system has these "schools" references
```

Comment:
No idea why my editor did this, but some setting is feature file aware and auto-aligns and inserts bars